app/models/duty_calculator.rb

```diff
@@ -57,4 +57,18 @@ def additional_duty_rows
         acc << option_klass.new(measure, user_session, []).option
       end
   end
+
+  def default_options
+    return [waiver_option] if user_session.gb_to_ni_route?
+
+    []
+  end
```

Comment:
oh, interesting so it's only on GB to NI route.

Reply:
Yarp